### ISSUE TITLE
Consider making the ad ID a number

### ISSUE THREAD

**Issue by andyleiserson** [patcg/ppa-api#6]
@benjaminsavage suggested that we make this a number.  Let's open an issue on that.

_Originally posted by @martinthomson in https://github.com/private-attribution/api/pull/1#discussion_r1746489586_

**Comment by csharrison** [comment]
I believe this issue is not relevant anymore, since I can't find the ad ID in the current spec. Shall we close it?

**Comment by andyleiserson** [comment]
Yes, we can close this. The ad ID was replaced with filter data.
